### PR TITLE
minidlna: remove panasonic video grouping by directory patch

### PR DESCRIPTION
**Pull request by jcadduono**
Unfortunately this is breaking sorting and causes some
issues with starting streams on various non-Panasonic
clients. Tested on 5 different models of 2014-2017
Samsung Smart TVs and BubbleUPnP for Android.
Removing this patch fixes sorting by filename and
clients no longer sometimes fail to load the streams.

Signed-off-by: James Christopher Adduono <jc@adduono.com>
-------------------------------

Maintainer: @medaved (so i was told)
Compile tested: armhf, R7800, LEDE GCC 7.2.0
Run tested: LEDE 4967 (master), minidlna 1.2.0 and minidlna 1.2.1 (in PRs)

Description:
Included in commit message above.

Perhaps this patch can be revisited and modified to be compatible with other clients?

### CODE REVIEW

**Comment by diizzyy** [comment]
LGTM

**Comment by diizzyy** [comment]
Can you please rephrase it to remove instead of kill?

**Comment by antonlacon** [comment]
LGTM too.
